packages/aws-cdk/lib/init-templates/app/csharp/src/%name.PascalCased%/%name.PascalCased%Stack.template.cs

```diff
@@ -4,7 +4,7 @@ namespace %name.PascalCased%
 {
     public class %name.PascalCased%Stack : Stack
     {
-        public %name.PascalCased%Stack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
+        public %name.PascalCased%Stack(Construct scope, string id) : base(scope, id)
```

Comment:
@RomainMuller pointed out that this would not be a good practice earlier so please hang on until he gets a chance to take a look

Reply:
Yes - the best practice is for Stack subclasses fo accept a `StackProps` parameter and forward it to the base. I wouldn't encourage users to omit this by encoding it in the template.